### ISSUE TITLE
Voice problem after 2.2.56 and Windows version after 2.2.59 (same as issue  #230)

### ISSUE THREAD

**Issue by BandekoCratermaker** [kauffman12/EQLogParser#231]
Have a non-technical guildie that I've been trying to help debug his recent issues.
2.2.56 Everything works fine
2.2.59 Voices no longer work, wav files play ok.  Tried different voices, test mode on voice, nothing.
2.2.62 He gets error trying to install "This program does not support the version of Windows your computer is running" as reported in issue #230. so will keep eye on that thread.

His setup
Win 11 Pro version 24H2  OS Build 26100.2605  
Feature Experience pack 1000.26100.36.0
.Net update 8,0.11 installed.

**Comment by kauffman12** [comment]
Is that an AMD system like in the other bug?

**Comment by BandekoCratermaker** [comment]
I'll find out

**Comment by BandekoCratermaker** [comment]
Intel i5 9600K, 16gb of ram, 64-bit

**Comment by kauffman12** [comment]
Yeah I'm not sure what the problem was with .62. Having [esr-sudo](https://github.com/esr-sudo) manually re-install it fixed their problem. I published a 63 from scratch just in-case something was corrupted somehow.

I guess share this link to have them install 63 and see what happens.  But maybe also delete the C:\Program Files\EQLogParser folder. That way it will have to install everything.

https://github.com/kauffman12/EQLogParser/raw/master/Release/EQLogParser-install-2.2.63.exe

The voice issue I don't understand. If they can play voices from Windows itself I don't see why they wouldn't work from the parser. Just make sure they test from the new speech settings. This one under Time & Language:

![image](https://github.com/user-attachments/assets/b736456c-d6b5-443b-9eb1-c137ea746743)


It's not the old one under text to speech. If their UI looks like the one below it's the wrong one.

![image](https://github.com/user-attachments/assets/cf49fc07-649f-49e2-b4de-1d5e9470dbea)

**Comment by BandekoCratermaker** [comment]
Uninstalled and Deleted Program Files folder. Installed .63, no more windows error there.  Voices still AFK from EQLP but do get voice from the Time & Language menu.  I did some light searching, There isn't a registry config that I should be wiping out?    When I uninstalled/delete and reinstall for him, his character/triggers configs are already there.       And always, thank you for all your work for the EQ community!

**Comment by kauffman12** [comment]
The settings are all under AppData so they wouldn't be touched by doing that. And there's really nothing in the settings that should break testing the voice when you select a different one from the menu.

If you can have them install this test build and then get their log file. See if they can just open it from the Tools -> View Error Log and copy/paste the contents.

https://github.com/kauffman12/EQLogParser/raw/refs/heads/master/Release/test/EQLogParser-install-2.2.63-test.exe

**Comment by BandekoCratermaker** [comment]
2024-12-31 13:11:40,219 [1] INFO EQLogParser.App - Detected OS Version: Microsoft Windows NT 6.2.9200.0
2024-12-31 13:11:40,239 [1] INFO EQLogParser.App - EQLogParser: 2.2.63, DotNet: 8.0.11
2024-12-31 13:11:40,272 [1] INFO EQLogParser.MainWindow - Window Pos (0, 0) | Window Size (1920, 1032)
2024-12-31 13:11:46,852 [1] INFO EQLogParser.AudioManager - Init Audio Manager
2024-12-31 13:11:46,902 [1] INFO EQLogParser.AudioManager - Using audio device: Headset Earphone (Logi Zone 750).
2024-12-31 13:11:46,932 [1] INFO EQLogParser.AudioManager - Adding speech synthesizer for d0ef2915-113f-4e9c-ba5b-574bac446505 and voice Microsoft David.
2024-12-31 13:11:46,933 [1] WARN EQLogParser.AudioManager - Failed Windows Version Check
2024-12-31 13:11:46,939 [1] INFO EQLogParser.AudioManager - Adding speech synthesizer for 8ddecc7e-7b7c-4b14-8eb2-86e26bad4862 and voice Microsoft David.
2024-12-31 13:11:46,940 [1] WARN EQLogParser.AudioManager - Failed Windows Version Check
2024-12-31 13:11:46,940 [1] INFO EQLogParser.AudioManager - Adding speech synthesizer for 0d89f1b9-20cc-400f-a975-dd75004f7f55 and voice Microsoft David.
2024-12-31 13:11:46,940 [1] WARN EQLogParser.AudioManager - Failed Windows Version Check
2024-12-31 13:11:46,940 [1] INFO EQLogParser.AudioManager - Adding speech synthesizer for 33885827-2103-4059-9bf5-2f2b9939e146 and voice Microsoft David.
2024-12-31 13:11:46,941 [1] WARN EQLogParser.AudioManager - Failed Windows Version Check
2024-12-31 13:12:21,488 [1] INFO EQLogParser.MainWindow - Selected Log File: D:\Games\Everquest\Logs\eqlog_Ravwolf_xegony.txt
2024-12-31 13:12:23,107 [1] INFO EQLogParser.MainWindow - Finished Loading Log File in 2 seconds.
2024-12-31 13:12:34,825 [1] WARN EQLogParser.AudioManager - Testing Speech: GUILD M O T D UPDATED  Reed it.
2024-12-31 13:12:34,826 [1] WARN EQLogParser.AudioManager - Failed Windows Version Check

**Comment by BandekoCratermaker** [comment]
but David does work from the Time & Language settings screen.

**Comment by BandekoCratermaker** [comment]
Didn't have him check other voices this time, but none worked after .56

**Comment by kauffman12** [comment]
It says they're using Microsoft Windows NT 6.2.9200.0 which is Windows 8 or 2012 Server. The speech synthesizer APIs are for windows 10. It's not even trying to use it because of the OS version check. 

If I had to guess they need to right-click on EQLogParser.exe and go to Properties. And Turn OFF this setting:

![image](https://github.com/user-attachments/assets/4c9b4a0e-5d9d-446a-98a0-b2e7c16c5a82)

**Comment by BandekoCratermaker** [comment]
That was it, someone else had been 'helping' him and they had set compat mode.  Didn't even cross my mind that was a possibility.   Thank you from me, and Thank You from him!

**Comment by kauffman12** [comment]
Good to hear.
